.gitignore

```diff
@@ -1,8 +1,8 @@
 *.iml
-.gradle
-/local.properties
-/.idea
+.gradle/
+local.properties
+.idea/
 .DS_Store
-/build
-/captures
+build/
+captures/
```

Comment:
Is there any reason to do that?

Reply:
I've had some issues where .gitignore doesn't detect /build (but detects build/), so eh...